app/src/hooks/useGraphMarketsFromQuestion.tsx

```diff
@@ -0,0 +1,66 @@
+import { useQuery } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+import { useEffect, useState } from 'react'
+
+import { Status } from '../util/types'
+
+const query = gql`
```

Comment:
so i saw that we have almost the same useGraphMarketFromQuestion hook that just has id as response instead of id and collateral...so i don't know if it's maybe better idea to add one extra field in that query than to create new file just for one more piece of data...and query name should be changed not to be same as the other one...
![Screenshot 2020-10-29 at 10 47 06](https://user-images.githubusercontent.com/33226956/97552784-f9d24400-19d4-11eb-8b67-4577c396068c.png)

Reply:
@Mi-Lan , the query is similar, but the purpose of each function is different logically.
So I created another similar one.

Reply:
@liaojikunwork ok i just thought that this approach is not according to DRY principle and to maybe handle that difference within the one hook if it's 90% same code but i guess you already thought about it so all is good

Reply:
@Mi-Lan , let me remove `useGraphMarketIdFromQuestion`